### ISSUE TITLE
Babel, svelte-jsoneditor >0.3.5 error: "You may need an additional loader to handle the result of these loaders."

### ISSUE THREAD

**Issue by dranitski** [josdejong/svelte-jsoneditor#38]
For some reason `svelte-jsoneditor@>0.3.5` crashes in react starter app just after being imported:

Steps to reproduce:
- `npx create-react-app my-app`
- `cd my-app`
- `yarn add svelte-jsoneditor@0.3.5`
- add this line to `src/App.js`: `import {JSONEditor} from 'svelte-jsoneditor/dist/jsoneditor.js';`
- run npm start
- see the error:
```
Failed to compile
./node_modules/svelte-jsoneditor/dist/jsoneditor.js 13423:12
Module parse failed: Unexpected token (13423:12)
File was processed with these loaders:
 * ./node_modules/react-scripts/node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
|     label: "descending"
|   }];
>   let f = u?.filter?.path ? I(u.filter.path) : null,
|       p = u?.filter?.relation ? h.find(e => e.value === u.filter.relation) : null,
|       m = u?.filter?.value || "",
This error occurred during the build time and cannot be dismissed.
```

I've noticed downgrading to svelte-jsoneditor@0.3.4 seems fixing the issue for now.
Thanks in advance!

**Comment by josdejong** [comment]
Thanks for reporting, and for the clear reproduction steps. 

I think that optional chaining `?.` is giving an issue here, I started using that in the TransformWizard in v0.3.5. So far the bundle isn't transpiled and I'm happily using all available JavaScript features 😁. 

I suppose if we want to prevent this issue we'll have to configure babel and transpile the bundle to be compatible with old browsers. Help would be welcome!

**Comment by dranitski** [comment]
Seems like we need this:

> If you’re using Babel, @babel/plugin-proposal-optional-chaining, @babel/plugin-proposal-nullish-coalescing-operator are available.

I will try it in my current project and let you know if it will solve the issue.

**Comment by josdejong** [comment]
Yes indeed. And `svelte-jsoneditor` currently doesn't have babel transpilation in place but we could add that to the rollup config. That would solve it for all consumers of the library.

**Comment by josdejong** [comment]
I just tried out with the latest `create-react-app@5.0.0` (which comes with Webpack 5) which was released 8 days ago on Dec 14th, that also solves the issue.

**Comment by brianpos** [comment]
Is there an equivalent Vuejs (or nuxt version)?
Having a few issues similar to this and just checking its expected to work, or if im first...
(also new to VueJS so could be just me)

**Comment by josdejong** [comment]
> Is there an equivalent Vuejs (or nuxt version)?

Can you please ask that in a different issue/question so we can keep this issue focused on the babel/webpack transpilation issue?

**Comment by gerardosabetta** [comment]
Hey I'm also affected by this issue, I did some troubleshooting and it seems that webpack 4 uses "acorn": "^6.4.1". And that version does not support the optional chaining operator. So consumers of your library using webpack with the babel loader ignoring `node_modules` will be unable to import the library. Unfortunately migrating from webpack 4 to 5 is not possible for me. 
Are you aware of any work arounds at the moment

**Comment by JonnyLeiZhang** [comment]
> Hey I'm also affected by this issue, I did some troubleshooting and it seems that webpack 4 uses "acorn": "^6.4.1". And that version does not support the optional chaining operator. So consumers of your library using webpack with the babel loader ignoring `node_modules` will be unable to import the library. Unfortunately migrating from webpack 4 to 5 is not possible for me. Are you aware of any work arounds at the moment

What a coincidence, I'm facing the exactly same situation today.

**Comment by CarterFendley** [comment]
Also effecting me with Vuejs + their electron builder. Any update on the PR by @gerardosabetta?

**Comment by josdejong** [comment]
@CarterFendley you can follow all updates of the PR in the PR itself: #85

**Comment by josdejong** [comment]
Fixed now in `svelte-jsoneditor@0.3.56`
